README.md

```diff
@@ -76,3 +76,7 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
   [<img src="/art/matchit.gif" width=240>](https://dartpad.dev/482dbd401031ab9f74b0fa3de705ad2f)
 
 ## Dart Samples
+
+- [Pattern](http://dartpad.dev/211761dbebee7c170cee8bc98d2314bc) by [Ayush Nishad](https://github.com/ayushnishad) (15/3/2020)
```

Comment:
Change the name here from `Pattern` to `Pyramid Pattern`.

Reply:
done.